docs/getting-started-guides/ubuntu/automated.md

```diff
@@ -0,0 +1,253 @@
+
+First we need to download the credentials and client application to your local workstation:
+
+Create the kubectl config directory.
+
+```shell
+mkdir -p ~/.kube
+```
+Copy the kubeconfig file to the default location.
+
+```shell
+juju scp kubernetes-master/0:config ~/.kube/config
+```
+
+Fetch a binary for the architecture you have deployed. If your client is a
+different architecture you will need to get the appropriate `kubectl` binary
+through other means.
+
+```shell
+juju scp kubernetes-master/0:kubectl ./kubectl
+```
+
+Query the cluster.
+
+```shell
+./kubectl cluster-info
+Kubernetes master is running at https://52.15.104.227:443
+Heapster is running at https://52.15.104.227:443/api/v1/proxy/namespaces/kube-system/services/heapster
+KubeDNS is running at https://52.15.104.227:443/api/v1/proxy/namespaces/kube-system/services/kube-dns
+Grafana is running at https://52.15.104.227:443/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana
+InfluxDB is running at https://52.15.104.227:443/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb
+```
+
+Congratulations, you've now set up a Kubernetes cluster!  
+
+## Scale up cluster
+
+Want larger Kubernetes nodes? It is easy to request different sizes of cloud
+resources from Juju by using **constraints**. You can increase the amount of
+CPU or memory (RAM) in any of the systems requested by Juju. This allows you
+to fine tune th Kubernetes cluster to fit your workload. Use flags on the
+bootstrap command or as a separate `juju constraints` command. Look to the
+[Juju documentation for machine](https://jujucharms.com/docs/2.0/charms-constraints)
+details.
+
+## Scale out cluster
+
+Need more workers? We just add more units:    
+
+```shell
+juju add-unit kubernetes
+```
+
+Or multiple units at one time:  
+
+```shell
+juju add-unit -n3 kubernetes
+```
+You can also ask for specific instance types or other machine-specific constraints. See the [constraints documentation](https://jujucharms.com/docs/stable/reference-constraints) for more information. Here are some examples, note that generic constraints such as `cores` and `mem` are more portable between clouds. 
+
+```shell
+juju add-unit kubernetes --constraints="instance-type=c4.large" 
+juju add-unit kubernetes --constraints="cores=4,mem=32G" 
+```
+You can also scale the etcd charm for more fault tolerant key/value storage:  
+
+```shell
+juju add-unit -n2 etcd
+```
+
+## Tear down cluster
+
+If you want stop the servers you can destroy the Juju model or the
+controller. Use the `juju switch` command to get the current controller name:  
+
+```shell
+juju switch
+juju destroy-controller $controllername --destroy-all-models
+```
+This will shutdown and terminate all running instances on that cloud. 
+
+## More Info
+
+The code that stands up a Kubernetes cluster is done in the charm code. The charm is built from using a layered approach to keep the code smaller and more focused on the operations of Kubernetes.
```

Comment:
Suggest:

We stand up Kubernetes with open-source operations, or operations as code, known as charms. These charms are assembled from layers which keeps the code smaller and more focused on the operations of just Kubernetes and its components.